fish-in-the-sea/huxley-comics-inspector
# Feature/overview panel

**CHANGES**
Add the info popup where a user can see if the extension is in maintenance mode and others stats about each issue.

**HOW TO TEST**
```
$ npm install
$ npm run build
```
- Refresh the extension : chrome://extensions/
- Click on the icon to see the popup

![image](https://user-images.githubusercontent.com/90482181/161543111-9d9c96ed-6b3b-4968-aa37-5cb8ebf8e3cd.png)

![image](https://user-images.githubusercontent.com/90482181/161543718-f505a824-963d-4e7f-b73b-ef4b93f14124.png)
